rules/library.bzl

```diff
@@ -797,9 +798,13 @@ def apple_library(name, library_tools = {}, export_private_headers = True, names
                 objc_hdrs.append(umbrella_header)
             module_map = library_tools["modulemap_generator"](
                 name = name,
+                library_tools = library_tools,
                 umbrella_header = paths.basename(umbrella_header),
+                public_headers = objc_hdrs,
+                private_headers = objc_private_hdrs,
```

Comment:
This was breaking our internal build due to missing default values, cc @dierksen